<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainInterface</class>
 <widget class="QWidget" name="MainInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainInterface</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="WG_sb">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="TargetTester" name="WG_target_tester"/>
     <widget class="SettingUi" name="WG_setting"/>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TargetTester</class>
   <extends>QWidget</extends>
   <header>targettester.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SettingUi</class>
   <extends>QWidget</extends>
   <header>settingui.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
